google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/HeaderInterceptor.java

```diff
@@ -98,27 +114,20 @@ private void processHeader(Metadata metadata, TagContext tagContext) {
           measureMap.put(SPANNER_GFE_LATENCY, latency);
           measureMap.put(SPANNER_GFE_HEADER_MISSING_COUNT, 0L);
           measureMap.record(tagContext);
+
+          SpannerRpcMetrics.recordGfeLatency(latency, attributes);
```

Comment:
No. For OpenTelemetry metrics and Spans to works , customers would need to create OpenTelemetry object and register as global else it will be no-op. 
For Shim customers need not create opentelemetry object.